core/integrations/integrations.js

```diff
@@ -0,0 +1,139 @@
+﻿/**
+ * Provides helper functions for handling output integrations.
+ *
+ * Written By:
+ * 		Matthew Knox
+ *
+ * License:
+ *		MIT License. All code unless otherwise specified is
+ *		Copyright (c) Matthew Knox and Contributors 2015.
+ */
+
+var files                   = require.once('../files.js'),
+    fs                      = require('fs'),
+    path                    = require('path'),
+    integrationsLocation    = 'core/core_integrations',
+    cachedIntegrations      = null,
+    selectedIntegrations    = null,
+    started                 = false;
+
+exports.listIntegrations = function () {
+    if (cachedIntegrations) {
+        return cachedIntegrations;
+    }
+    var integ = files.filesInDirectory('./' + integrationsLocation),
+        list = [];
+    for (var i = 0; i < integ.length; i++) {
+        var p = path.join(__dirname, '../../'+ integrationsLocation, integ[i]);
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Infix operators must be spaced.](https://www.codacy.com/app/theprojectshop/Kassy/file/2530929735/issues/source?bid=3282987&fileBranchId=3283570#l27)